color_windows.go

```diff
@@ -0,0 +1,18 @@
+package color
+
+import (
+	"os"
+
+	"golang.org/x/sys/windows"
+)
+
+func init() {
+	// Opt-in for ansi color support for current process.
```

Comment:
Can we use `NO_COLOR` env var and `NoColor = true` global variable to opt-out?

https://github.com/fatih/color/pull/173#discussion_r1116624911

Reply:
the `NO_COLOR` env already changes the `NoColor` value. So we can just check for the `NoColor` variable and it'll cover everything.

Reply:
So I guess it is just a matter of adding

```golang
if NoColor {
  return
}
```

right? 😉

Reply:
The problem though is, it wouldn't work as expected. Imagine this situation:

1. You import color and enable `NO_COLOR`.
2. The `init()` is called, it check's `NoColor`, if it's false it will not set the configuration
3. After a while though, you set `NoColor = false`.
4. It still doesn't work, because `init()` will not run anymore. So we need a separate enviornment variable. `NoColor` is  working fine as it's now, but it wouldn't work opt-out.

Reply:
Regarding #2. init() is run after all var () blocks, so NoColor would be set before init() is called.